TelligentLocalInstance/Initialize.ps1

```diff
@@ -281,4 +305,134 @@ function Write-Telligent
 	Write-Host
 }
 
+function Expand-Zip {
```

Comment:
I think we may still the custom version in places as Expand-Archive doesn't support extracting just a portion (e.g. just Web), although I'd prefer to use the inbuilt functions where possible.